data/projects/integrate-pyflakes-AST.md

```diff
@@ -0,0 +1,24 @@
+flake8 has many plugins which are commonly used in the python development community.  flake8 is a wrapper, similar to coala, that provides that use the python-enhanced AST to its plugins.  Instead of coala using the flake8 wrapper, which adds another level of complexity and uncertainty with regards to configuration, coala would like to directly load the python-enhanced AST and invoke the flake8 plugins.
```

Comment:
"... that provides the python-enhanced AST to its plugins." The "that use" is not needed.